src/test/java/com/nerodesk/AppTest.java

```diff
@@ -139,6 +139,36 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * Application can return file content in binary form.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void returnsBinaryContent() throws Exception {
+        final Base base = new MkBase();
+        final String name = "test.dat";
+        final byte[] content = new byte[]{'t', 'e', 's', 't'};
```

Comment:
I would suggest to use something less obvious here. Something like `{0x00, 0x0a, 0xff, 0xfe}`